pkg/orm/db_mysql.go

```diff
@@ -42,24 +42,25 @@ var mysqlOperators = map[string]string{
 
 // mysql column field types.
 var mysqlTypes = map[string]string{
-	"auto":            "AUTO_INCREMENT NOT NULL PRIMARY KEY",
-	"pk":              "NOT NULL PRIMARY KEY",
-	"bool":            "bool",
-	"string":          "varchar(%d)",
-	"string-char":     "char(%d)",
-	"string-text":     "longtext",
-	"time.Time-date":  "date",
-	"time.Time":       "datetime",
-	"int8":            "tinyint",
-	"int16":           "smallint",
-	"int32":           "integer",
-	"int64":           "bigint",
-	"uint8":           "tinyint unsigned",
-	"uint16":          "smallint unsigned",
-	"uint32":          "integer unsigned",
-	"uint64":          "bigint unsigned",
-	"float64":         "double precision",
-	"float64-decimal": "numeric(%d, %d)",
+	"auto":                "AUTO_INCREMENT NOT NULL PRIMARY KEY",
+	"pk":                  "NOT NULL PRIMARY KEY",
+	"bool":                "bool",
+	"string":              "varchar(%d)",
+	"string-char":         "char(%d)",
+	"string-text":         "longtext",
+	"time.Time-date":      "date",
```

Comment:
Is it possible that users need "time.Time-date-precision"?

Reply:
The date saves the precision to the day, the format is: YYYY-MM-DD